### PR TITLE
EDA Settings Container Permissions Check - Backend and Unstyled

### PR DESCRIPTION
**Pull request by xinnanlisf**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/classes/PermissionSetAssignmentMapper_TEST.cls
Comment (MatthewBlanski) [inline force-app/main/default/classes/PermissionSetAssignmentMapper_TEST.cls:75]: This is an elegant solution to testing! I did not know you could do that!

File: force-app/main/default/classes/PermissionSetMapper.cls
Comment (MatthewBlanski) [inline force-app/main/default/classes/PermissionSetMapper.cls:73]: Why don't we return `!permissionSets.isEmpty()` or `permissionsSets.isEmpty() == false`?

File: force-app/main/default/classes/UserPermissionService.cls
Comment (MatthewBlanski) [inline force-app/main/default/classes/UserPermissionService.cls:71]: We can drop the else here and simply return true. The above statement is acting as a guard clause.

**Review by MatthewBlanski — APPROVED** [review]
The code is functional. I've left a few stylistic comments, but they are not requirements.

File: force-app/main/default/classes/PermissionSetAssignmentMapper_TEST.cls
Comment (xinnanlisf) [reply force-app/main/default/classes/PermissionSetAssignmentMapper_TEST.cls:75]: I just found out too!

**Comment by xinnanlisf** [comment]
Having trouble retrieving custom label. I might be able to manually add but adding translation files would be a pain. 

Also, dependency on container PR #1579. We could merge the changes from but since we don't have dependency on this PR. We could potentially wait until it is merged to 230.1.

**Review by m-kinkade — APPROVED** [review]

**Review by MatthewBlanski — APPROVED** [review]

**Comment by spinkelman** [comment]
QA Passes - Merging-Closing WI

**Pull request merged**
